[ENHANCEMENT] brief primer on MCP  Model Context Protocol (#316)
huggingface/agents-course: units/en/unit1/tools.mdx

In the [Actions](actions) section, we will learn more about how an Agent can **Call** this tool we just created.

> ### Model Context Protocol (MCP) a unified Tool interface

Reviewer: ```suggestion
### Model Context Protocol (MCP): a unified tool interface.
```

In the [Actions](actions) section, we will learn more about how an Agent can **Call** this tool we just created.

### Model Context Protocol (MCP) a unified Tool interface

Model Context Protocol (MCP) is an **open protocol** that standardizes how applications **provide tools to LLMs**.
MCP provides:

- A growing list of pre-built integrations that your LLM can directly plug into
- The flexibility to switch between LLM providers and vendors
- Best practices for securing your data within your infrastructure

> This means that **any of the frameworks that implement MCP can use tools that are defined in the MCP**.
> Getting rid of the need to reimplement the same tool interface for each framework.

Reviewer 2: ```suggestion
This means that **any framework implementing MCP can leverage tools defined within the protocol**, eliminating the need to reimplement the same tool interface for each framework.
```
